WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListViewModel.swift

```diff
@@ -1,29 +1,49 @@
 import Combine
 import Foundation
+import Networking
```

Comment:
Thanks for letting me know about this, I was concerned about this, but since I found that we already import it in other places in `WooCoommerce`, I thought that it's fine 🤷.

I updated the code now following the second approach, as we need to import this only in a single file 4e7c8e3